tests/CMakeLists.txt

```diff
@@ -72,11 +72,11 @@ if (WITH_UNITTESTS)
     target_link_libraries(comparators_tests ${ENGINE_LDFLAGS} ${ENGINE_WRITE_LIBS} ${CPPUNIT_LIBRARIES} cppunit)
     add_test(NAME columnstore:comparators_tests COMMAND comparators_tests)
 
-    add_executable(patterns_test patterns_test.cpp)
-    add_dependencies(patterns_test googletest mpark_patterns)
-    target_include_directories(patterns_test PUBLIC ${ENGINE_COMMON_INCLUDES} ${ENGINE_BLOCKCACHE_INCLUDE} ${ENGINE_PRIMPROC_INCLUDE} )
-    target_link_libraries(patterns_test ${GTEST_LIBRARIES})
-    gtest_add_tests(TARGET patterns_test TEST_PREFIX mpark_patterns:)
+    #add_executable(patterns_test patterns_test.cpp)
```

Comment:
This should be active.

Reply:
Will fix.

Reply:
It produces "internal compiler error" on my Rocky Linux 8 builds with gcc version 8.5.0 20210514. The library is not used anywhere else.

I will uncomment, of course, after I am done with your other notes.

Reply:
Much appreciated.

Reply:
Did that.

Reply:
Don't forget to uncomment this.